### PR TITLE
Fix smoke test when flashbox chain has no credits

### PR DESCRIPTION
**Pull request by tmpolaczyk**

### CODE REVIEW

File: test/util/payment.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export async function hasEnoughCredits(
     paraApi: ApiPromise,
     paraId: ParaId,
     blocksPerSession: bigint,
+    // TODO: minSessionRequirement should be 2 if the chain had collators in the previous session, and 1 otherwise
```
Comment (tmpolaczyk) [inline test/util/payment.ts:20]: Not sure if there is an easy way to check this, we would need to get the assignment from the previous session

Comment (girazoki) [reply test/util/payment.ts:20]: I think it's one in either case no?¿

Comment (tmpolaczyk) [reply test/util/payment.ts:20]: I was thinking about this if condition: https://github.com/moondance-labs/tanssi/blob/443f13c0cc2e4ee44c89273c6bdfbfd61181a62c/runtime/dancebox/src/lib.rs#L795

I guess it depends on when we check this, if we look at the pending assignment we may need 2, if we check the current assignment we need 1

Comment (girazoki) [reply test/util/payment.ts:20]: This is done in the collator-assignation, and we always (AFAIK) do it with pending. Why do we need 2? We are charging immediatly the collator-assignment for the **next session**. I remember discussing this with @fgamundi

**Review by girazoki — APPROVED** [review]

**Comment by girazoki** [comment]
I would not overcomplicate the test. Let's leave it as is, and let's rethink about this in the future

File: test/util/payment.ts (hunk 1 above)
Comment (fgamundi) [reply test/util/payment.ts:20]: Yes, it's done with pending. So it goes that if you want to be assigned for the next session, and you are already set to collate for session about to start, you need enough to produces the blocks for this + next session. But the session credits checked to be assigned is always only 1

**Pull request merged**